src/lib/y2storage/planned/lvm_vg.rb

```diff
@@ -77,11 +73,22 @@ def self.from_real_vg(real_vg)
       # @param pvs               [Array<String>] Name of partitions to be used as physical volumes
       def initialize(volume_group_name: nil, lvs: [], pvs: [])
         initialize_has_size
-        @volume_group_name = volume_group_name || DEFAULT_VG_NAME
+        @volume_group_name = volume_group_name
         @lvs = lvs
         @pvs = pvs
       end
 
+      # Initializes the object taking the values from a real volume group
+      #
+      # @param real_vg [Y2Storage::LvmVg] Real volume group
+      def initialize_from_real_vg(real_vg)
+        @extent_size = real_vg.extent_size
+        @total_size = real_vg.total_size
+        @pvs = real_vg.lvm_pvs.map { |v| v.blk_device.name }
```

Comment:
I wonder if this could be potentially dangerous at some point since names can change as we create and destroy partitions. We will find out. :wink: